ingredient/ingredient-storage/src/functions.ts

```diff
@@ -42,4 +42,28 @@ export class StorageUtils extends BaseUtility {
         }
         return key
     }
+
+    public async get_primary_connectionstring(name: string, rg: string | null = null) : Promise<string> {
+        let key: string = ""
+
+        key = await this.get_primary_key(name, rg);
+        
+        let accountName: string = this.create_resource_name();
```

Comment:
you're getting a key for a storage account via the "name" variable, but then creating the account name here based on the current package. These should be the same.

Reply:
Thank you I’ll fix